### PR TITLE
Mattherndon/falldinner2018

### PR DESCRIPTION
**Pull request by amicableskeptic**
Adding fall dinner 2018 recap.

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -128,6 +128,25 @@ <h2 class="title-margin-bottom">News &#x26; Events</h2>
...
+                   <img class="news-img" src="images/Cornell20180915Fencing.jpg" alt="Coaches and Guests" />
+
+                   <p>September 15, 2018 at the New York Athletic Club</p>
+
+                   <p>Alumni, Olympians, World Team members and Olympic Coaches wined, dined and some even fenced the night away at our 3rd Annual CFA Fall Dinner. In what portends to be a winning season ahead, Head Coach Daria Schneider, the evening&#x2019;s MC,  introduced us to Cornell&#x2019;s newest coach, Omar Elgeziry. A 2016 Olympian himself, he was previously assistant coach at Air Force for 3 seasons and coached 11 athletes into qualifying positions in the NCAA. He also spent one year as the national epee coach for the USA Pentathlon. We welcome him to the Cornell family.</p>
+                   
+                   <p>We also said our farewell to Nicole Ross, who is taking a leave of absence to prepare for qualification to the 2020 Olympic Team.  As current world champion, we know she will continue to build on her success. She spoke about her enjoyment with working with the team and being a member of the Cornell fencing family. We wish her all the best! </p>
```
Comment (NickHeiner) [inline app/index.html:140]: Let's try to be consistent on spaces after the period. The previous paragraph had a 1-spacer, and this paragraph had a 2-spacer.

File: app/index.html [hunk 2]
```diff
@@ -128,6 +128,25 @@ <h2 class="title-margin-bottom">News &#x26; Events</h2>
...
+                   
+                   <p>Board Member and past President of the CFA, Valerie Leimer gave an impassioned speech revealing her personal reason for contributing so generously to the reinstatement effort. We ask that all fencing alumni look for their reason, their most personal &#x201C;why&#x201D;, to contribute towards this goal. A unified Men&#x2019;s and Women&#x2019;s team brings more than just all IVY titles, but a camaraderie that is unique to NCAA sports.</p>
+                   
+                   <p>We welcomed our honored guests Greg Massialas and Buckie Leach who shared stories of their time at Cornell as well as their experience working with both Daria and Nicole. They wished for the continued success of the Cornell fencing program and included their support of a future reinstated men&#x2019;s team.</p>
```
Comment (NickHeiner) [inline app/index.html:144]: Let's be consistent about whether it's `Men's` or `men's`.

**Comment by NickHeiner** [comment]
I left a few comments, but broadly LGTM. Feel free to squash merge and deploy yourself after taking a look at them. Thanks!

File: app/index.html (hunk 1 above)
Comment (amicableskeptic) [reply app/index.html:140]: This is likely due to me direct copying text from Nina. Luckily HTML will kindly mash all extra spaces down to one so only code viewers will notice it :)

File: app/index.html (hunk 2 above)
Comment (amicableskeptic) [reply app/index.html:144]: I started doing this and it turns out there are a lot of edge cases spread throughout the page. I am just going to leave it all as is and we can come up with a consistent capitalization guide when we do the website update.

**Pull request merged**